src/packagedcode/pypi.py

```diff
@@ -295,9 +301,11 @@ def parse_setup_py(location):
         version=setup_args.get('version'),
         description=description or None,
         homepage_url=setup_args.get('url') or None,
+        download_url=download_url or None,
         parties=parties,
         declared_license=declared_license,
-        keywords=other_classifiers,
```

Comment:
I get your point to use keywords for keywords, but we would also need to keep PyPI classifiers as keywords too. The important thing is that the Package model is shared by all the packages (pypi, maven, npm, etc)... so we cannot treat classifiers specially... just as plain keywords... so IMHO `keywords=other_classifiers+keywords`